src/features/ProductPage/json-ld.ts

```diff
@@ -0,0 +1,56 @@
+import { ProductProps } from 'features/ProductPage/Product/Product';
+import { ProductJsonLdProps } from 'next-seo';
+import { ReviewList } from 'types/review';
+
+export type GetProductJsonLdPropsArgs = Pick<ProductProps, 'product'> & {
+  reviewsIoReviewList: ReviewList | null;
+  trustpilotReviewList: ReviewList | null;
+};
+
+export const getProductJsonLdProps = ({
+  product,
+  reviewsIoReviewList,
+  trustpilotReviewList
+}: GetProductJsonLdPropsArgs): ProductJsonLdProps => {
+  const reviewsForJsonLd = reviewsIoReviewList ?? trustpilotReviewList;
+
+  let publisher: string;
+  if (reviewsIoReviewList) {
```

Comment:
See other note: I believe this could be a `publisher` prop on the `ReviewList` type, or something similar.